src/sentry/static/sentry/app/components/group/pluginActions.tsx

```diff
@@ -13,22 +13,28 @@ import {Group, Organization, Plugin, Project} from 'app/types';
 import withApi from 'app/utils/withApi';
 import withOrganization from 'app/utils/withOrganization';
 
+type TrelloIssue = {
```

Comment:
@EvanPurkhiser nor every Plugin is Trello hahah

Reply:
uhg Im dumb LOL